packages/react-reconciler/src/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -35,10 +35,11 @@ describe('ReactIncrementalErrorHandling', () => {
     return str && str.replace(/in .+? \(at .+?:\d+\)/g, '').trim();
   }
 
-  function stripStackTrace(obj) {
-    let stripped = [...obj];
-    stripped[0].prop = removeStack(stripped[0].prop);
-    return stripped;
+  function stripStackTrace(compSpan) {
+    return [{
+      ...compSpan[0],
+      prop: removeStack(compSpan[0].prop),
+    }];
```

Comment:
If there is (incorrectly) more than one child, this will ignore it. Let's make sure we don't "lose" any input children?

Reply:
Gotcha. Let me see what I can come up with.